.github/workflows/main.yml

```diff
@@ -1,8 +1,9 @@
+#name: "Generate Website Data"
```

Comment:
Rm commented-out line

Reply:
Updated the comment to explain what's going on.  This is a transitional state.  After merging into the master branch, GitHub will be aware of two workflows and then I can make the required changes on a branch to turn this workflow into something else.